docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,28 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.8 - December 24, 2024
+
+### Enhancements
+
+- Assessment and Schema Analysis Reports
+  - [assess-migration](../reference/assess-migration/) and [analyze-schema](../reference/schema-migration/analyze-schema/) now accept a new flag `--target-db-version`, which allows the user to specify the version of YugabyteDB they wish to assess/analyze against. It defaults to the latest stable release version (currently {{<yb-version version="stable">}}).
+  - Presence of advisory locks, XML functions, and system columns in DDLs will now be detected and reported in assessment and schema analysis reports.
+  - Presence of [large objects](https://www.postgresql.org/docs/current/largeobjects.html) (and its functions) in DDLs/DMLs is now detected and reported in assessment and schema analysis reports.
+  - In the Schema analysis report (html/text), a few field names have been modified to improve readability. Invalid Count -> Objects with Issues, Total Count -> Total Objects, Valid Count -> Objects without Issues. The logic of determining when an object is considered to have issues or not has also been improved.
+  - Stop reporting Unlogged tables as an issue in assessment and schema analysis reports by default, as it has been fixed in YugabyteDB {{<release "2024.2.0.0">}}.
```

Comment:
"it" is https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#unlogged-table-is-not-supported so maybe link to it
"Stop reporting unlogged tables as an issue in assessment and schema analysis reports by default, as [the issue](link) has been fixed in YugabyteDB {{<release "2024.2.0.0">}}."
Is unlogged really supported in 2024.2? https://docs.yugabyte.com/stable/api/ysql/the-sql-language/statements/ddl_create_table/#unlogged

Reply:
Yeah, the subtlety is that it's not "fixed" as such, just ignored... But from voyager's perspective we decided not to report it as an "issue" anymore because the user doesn't really have to manually intervene. 

We can perhaps phrase it as "... as the UNLOGGED option no longer results in a syntax error in YugabyteDB 2024.2.0.0" ?

Reply:
I've rephrase id as @makalaaneesh said.